junit5/src/main/java/sunstone/api/WithAwsCfTemplates.java

```diff
@@ -0,0 +1,19 @@
+package sunstone.api;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Inherited;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * Deploy CloudFormation template
+ *
+ * Deployed as a stack in BeforeAll and stack is deleted in AfterAll
+ */
+@Retention(RetentionPolicy.RUNTIME)
+@Target({ElementType.TYPE})
+@Inherited
+public @interface WithAwsCfTemplates {
```

Comment:
Why are there both `WithAwsCfTemplate` and `WithAwsCfTemplates`?

Reply:
`WithAwsCfTemplates` makes `WithAwsCfTemplate` repetable. So I can deploy multiple templates.